### PR TITLE
FFT presentation

### PR DESCRIPTION
**Pull request by tristpinsm**

### CODE REVIEW

File: 05_fft.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
+---
```
Comment (jrs65) [inline 05_fft.md]: Rename fill to `06_fft.md` (lecture 5 already exists)

File: 05_fft.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+$F(k)$ is the Fourier transform of $f(x)$. Because it is common to take Fourier
+transforms of time-series data, we often refer to $f(x)$ as being in the "time
+domain" as $F(k)$ its representation in the "frequency domain". There is no
+fundamental difference between the two, the inverse transform has the same form
+and is lossless.
+$$
+  F(k) = \int_{-\infty}^{\infty} dx~e^{-2 \pi i k x} f(x) \text{,}
+$$
+$$
+  f(x) = \int_{-\infty}^{\infty} \frac{dk}{2 \pi}~e^{2 \pi i k x} F(k) \text{.}
```
Comment (jrs65) [inline 05_fft.md]: With frequencies rather than angular frequencies, there isn't the 2 pi in the denominator.

File: 05_fft.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+## Aliasing
+What if the sampling rate is too low to capture the highest frequencies?
+$$
+  \Delta t \lt \frac{1}{2 \Delta\nu}
+$$
+It becomes impossible to distinguish between certain pairs of frequencies.
+
+![](./plots/fft_alias.png){ width=50% }
+
+## Aliasing in Ringmaps
```
Comment (jrs65) [inline 05_fft.md]: It's not clear that they now that ring map imaging is an FFT yet. You could maybe move this until the end.

**Review by jrs65 — APPROVED** [review]
Material looks good. I had a few comments, and some of the slides have too much on them such that they spill off the page.

**Comment by tristpinsm** [comment]
ok, thanks for taking a look! I'm not sure what to do about the material spilling off the screen. It all fits in my browser/screen.

**Comment by jrs65** [comment]
At default zoom level stuff was spilling off, but if I zoom out it's all good. Not sure if there's anything that can actually be done about that one.

**Pull request merged**